CHANGELOG.rst

```diff
@@ -1,3 +1,24 @@
+Version 2.5.0
+-------------
+
+:Date: June 06, 2018
+
+* `@ericholscher <http://github.com/ericholscher>`_: Fix Build: Convert md to rst in docs (`#4199 <https://github.com/rtfd/readthedocs.org/pull/4199>`_)
```

Comment:
Links are in the old format, we should update the common submodule for this to apply https://github.com/rtfd/common/pull/8